<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWidget</class>
 <widget class="QWidget" name="FormWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(30, 30, 30); color:white;</string>
  </property>
  <widget class="QLineEdit" name="InputEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white; border-radius:5px; background-color:black; </string>
   </property>
  </widget>
  <widget class="QLabel" name="InputLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Input</string>
   </property>
  </widget>
  <widget class="QLabel" name="KnowledgeLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Knowledge</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="KnowledgeEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid white; border-radius:5px; background-color:black; </string>
   </property>
  </widget>
  <widget class="QRadioButton" name="WordPartRadio">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Word Part</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="WordRadio">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Word</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="SentenceRadio">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sentence</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="OtherRadio">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>180</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Other</string>
   </property>
  </widget>
  <widget class="QLabel" name="LowerBoundLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lower bound</string>
   </property>
  </widget>
  <widget class="QLabel" name="UpperBoundNumber">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>410</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black; border: 2px dashed rgb(255, 55,55); border-radius: 5px; color:rgb(255, 255, 55);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TargetNumber">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black; border: 2px dashed rgb(55, 55, 255); border-radius: 5px; color: rgb(55,255,255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LowerBoundNumber">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>410</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black; border: 2px dashed rgb(105, 255, 105); border-radius: 5px; color: rgb(255, 50, 10);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="UpperBoundLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Upper bound</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Target</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SubmitButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(20,20,80); border-radius:5px; border: 2px solid rgb(100,255,255);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GroupButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(20,20,80); border-radius:5px; border: 2px solid rgb(100,255,255);</string>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
